typescript/utils/src/types.ts

```diff
@@ -69,6 +69,7 @@ export type S3Checkpoint = {
 export type CallData = {
   to: Address;
   data: string;
+  value: BigNumber;
```

Comment:
optional?

Reply:
I fixed it everywhere so it's ok to make it required?

Reply:
It's technically a breaking change and you force every consumer to do `BigNumber.from(0)` which is kind of annoying
afaict we dont have a motivating non-zero value use case planned atm

Reply:
fixed